common/lib/capa/capa/inputtypes.py

```diff
@@ -319,8 +319,14 @@ def _get_render_context(self):
             'msg': self.msg,
             'response_data': self.response_data,
             'STATIC_URL': self.capa_system.STATIC_URL,
+            'describedby': '',
         }
 
+        if self.response_data.get('description_ids'):
+            context.update(
+                {'describedby': 'aria-describedby="{}"'.format(self.response_data.get('description_ids'))}
```

Comment:
@muhammad-ammar if we are passing the description_ids as a value of `aria-describedby` then we don't need to maintain an extra attribute in `response_data `